.github/workflows/reviewdog-workflow.yml

```diff
@@ -113,6 +116,26 @@ jobs:
           # Ignore DL3005-"Do not use apt-get upgrade or dist-upgrade"
           hadolint_ignore: DL3005
 
+  eslint:
```

Comment:
Does that mean we are now running `eslint` twice once as part of the nms workflow and ones in the reviewdog workflow?

Reply:
That is correct. However, I do not see an issue with that, as it will be run in parallel and thus not prolong the CI execution.
I would see this as first step in part of a larger consolidation of the CI, in this part of linters and formatters.
Anyhow, for now, it brings the advantage that people opening a PR see in the PR conversation, what issues the Linter shows, instead of only having those hidden in the run logs of the action.